app/python/views.py

```diff
@@ -83,29 +85,44 @@ def create_new_challenge():
 
 @python.route('api/v1/python-challenges/<id>', methods=['PUT'])
 def update_challenge(id):
-    challenge_data = loads(request.form.get('challenge'))['challenge']
+    challenge_data = request.form.get('challenge')
+    if challenge_data != None: challenge_data = loads(challenge_data)['challenge']
     save_to = "public/challenges/"  #general path were code will be saved
+
     req_challenge = PythonChallenge.query.filter_by(id=id).first()
-    
+
     if req_challenge is None:   #case id is not in database
         return make_response(jsonify({"challenge":"there is no challenge with that id"}),404)
 
     response = PythonChallenge.to_dict(req_challenge).copy()   #start creating response for the endpoint
-    
+    response.pop('best_score', None)
```

Comment:
it does not seem a very good idea to take the best score

Reply:
you are right, postman shows that best_score needs to be returned in response